src/configure/extract_from_c.ml

```diff
@@ -0,0 +1,118 @@
+(*
+ * Copyright (c) 2016 whitequark.
+ *
+ * This file is distributed under the terms of the MIT License.
+ * See the file LICENSE for details.
+ *)
+
+let getenv ~default name =
+  try Sys.getenv name
+  with Not_found -> default
+
+let nsplit sep str =
```

Comment:
I think you'll agree with me that Str is badly designed and should be deprecated, if so, why use it if I already wrote code without it?